descriptors/ModuleDescriptor-template.json

```diff
@@ -141,6 +141,10 @@
     {
       "id": "permissions",
       "version": "5.6"
+    },
+    {
+      "id": "permissions-users",
```

Comment:
I assume permissions and permissions-users interfaces cannot be presented at the same time on platform. Probably, optional interface should be used here https://github.com/folio-org/okapi/blob/master/doc/guide.md#optional-interfaces.

Reply:
permissions is always present (Eureka uses it as well), but `permissions-users` is definitely should be moved to optional as okapi won't have it

Reply:
It was done thanks